### PR TITLE
Raise exception when CSV file with XLS extension is detected

### PR DESCRIPTION
**Pull request by iriberri**
Fixes https://github.com/CartoDB/cartodb/issues/6421 

* [x] Make sure that excel2csv specs are included in the Makefile

https://github.com/CartoDB/cartodb-management/issues/4383

### CODE REVIEW

**Comment by iriberri** [comment]
Please, @rafatower, take a look when you can :-)

**Comment by rafatower** [comment]
Ok, but even better with a test, should be easy to write

**Comment by iriberri** [comment]
The current tests for this should be actually failing as far as I've seen, but they aren't. Taking a look at them.

File: services/importer/lib/importer/excel2csv.rb [hunk 1]
```diff
@@ -32,13 +32,12 @@ def run
...
+          file_mime_type = stdout.read.delete("\n")
           job.log "Can't get the mime type of the file" unless process.value.to_s =~ /exit 0/
+          # CSV files with XLS extensions are considered malformed files
+          raise CartoDB::Importer2::MalformedXLSException.new if file_mime_type == "text/plain"
         end
-
-        # Take into account that here should come or csv files with xls extensions or xls documents
-        file_format = (@file_mime_type == "text/plain") ? "-f csv" : ""
-        in2csv_command_line = %Q[in2csv #{filepath} #{file_format} | #{in2csv_warning_filter} | #{newline_remover_path} > #{converted_filepath}]
+        in2csv_command_line = %Q[in2csv #{filepath} | #{in2csv_warning_filter} | #{newline_remover_path} > #{converted_filepath}]
```
Comment (iriberri) [inline services/importer/lib/importer/excel2csv.rb]: Just for the record, `#{in2csv_warning_filter}` contains double quotes.

**Comment by rafatower** [comment]
IIRC the test file needs to be added to the `Makefile` for CI to execute them, right?

**Comment by iriberri** [comment]
Yes! I confirmed with Systems. Will update the Makefile for the CI to run again here and then we'll see. :-)

**Comment by iriberri** [comment]
@luisbosque, I just made a spec addition into the Makefile. Could you please confirm that I won't be breaking anything or if there's something extra that needs to be done for it to run in CI? :heart:

**Comment by luisbosque** [comment]
AFAIK that's the right way to add it. Go ahead!

**Comment by iriberri** [comment]
Ready to be deployed then. Thanks!

**Pull request merged**